### ISSUE TITLE
Feature Request: Specify database/log directory

### ISSUE THREAD

**Issue by StevenDTX** [Jukari2003/Duplicate-Media-Finder#1]
I've been running this against a directory with 57,000 files for the last few days, and I didn't realize that the database/log directory would get enormous.  Mine is over 50GB and counting, and I'm only 25% through.

Would be great to set a full path to the database/log directory.  If its going to get huge, I would rather keep it on my NAS or something.

Thanks!

**Comment by Jukari2003** [comment]
Hey Steven,

When I initially published the script I accidentally left a debug setting on to "Keep Screenshots". You do not need these they are not used to do any further processing and are most likely culprit on why your logs are huge. Go into your database and delete all the screenshots - that should fix the issue. Also, you may want to download the latest version as there were some logging issues I just fixed. 

Hope this helps!
- Anthony

**Comment by StevenDTX** [comment]
> Hey Steven,
> 
> When I initially published the script I accidentally left a debug setting on to "Keep Screenshots". You do not need these they are not used to do any further processing and are most likely culprit on why your logs are huge. Go into your database and delete all the screenshots - that should fix the issue. Also, you may want to download the latest version as there were some logging issues I just fixed.
> 
> Hope this helps!
> 
>     * Anthony

Thanks for the reply.  Yes, the images are taking up all of the space.  

So, if I download the version, and start it again, it will pick up where it left off and not re-evaluate what its already done, correct?

**Comment by Jukari2003** [comment]
It depends, some of the issues I fixed targeted that very problem.... If you ran in "Log Only" mode, it will forget what files you already found.... (That should be fixed in the new update). If you ran in Rename Duplicates it will pick up the duplicate files upon running again. However, your Fingerprints and keys are still valid it should files you already have established "Skip Known Files" option. However, any duplicates that were previously identified will be lost under version 2.0. 

Long story short, I would make a copy of the log or make note of duplicates it found before you shut down your scan, but it will quickly bypass the files you already logged.
